README.md

```diff
@@ -1,8 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# OOP school library project
 
-> Description the project.
+> Project 1: add Person, Student, and Teacher classes.
+> Project 2: use the "Decorator" design pattern.
+> Project 3: set up associations.
+> Project 4: add basic UI.
+> Project 5: refactor your code.
+> Project 6: preserve data.
+> Project 7: unit tests.
```

Comment:
- [ ] As requested by the previous reviewers, let's make the `README.md` more professional. Kindly update the project description to something more descriptive of what the project is about instead of having a list of the project milestone ToDos. 👍🏾